IdentityCore/tests/mac/MSIDMacKeychainTokenCacheTests.m

```diff
@@ -724,6 +724,7 @@ - (void)testSaveAccount_whenAccountSaved_shouldSaveValidLastModInfo
     // check behavior if item had been written by a different process:
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->